training/train.py

```diff
@@ -56,30 +56,29 @@ def load_s3():
     raise Exception("S3 Connection Failed")
 
 
-def load_data(s3_client):
+def load_data(s3_client):  # pragma: no cover
```

Comment:
I would've preferred the arrangement of these functions into logical units rather than just simply breaking down for the sake of it, like we discussed but if you want feel free to create an issue for that in JIRA and put it in the next sprint.

Standard code review says it needs to be done before merging this one but I'm ok to let it slide if you can at least split the accuracy bits into a different file.

Reply:
@rootAvish I have created a JIRA issue: https://jira.coreos.com/browse/APPAI-160